.github/workflows/jira.yml

```diff
@@ -18,11 +18,10 @@ jobs:
     name: Jira sync
     steps:
     - name: Check if community user
-      if: github.event.action == 'opened'
```

Comment:
Why do we need to remove this?

Reply:
Yup sorry. I tagged you too early. I still had some test commits so that I could test the changes.

Reply:
I AM A SLAVE TO MY NOTIFICATIONS MICHAEL

Reply:
😂 I realized too late. I tried to remove you from review in the interim, but alas, I am not faster than the internet.